staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/encryptionconfig_test.go

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
The package name is `encryptionconfig`. It may be violating some convention, but I'm not entirely sure.